x-pack/plugins/synthetics/server/synthetics_service/service_api_client.ts

```diff
@@ -31,6 +31,20 @@ export interface ServiceData {
   license: LicenseGetLicenseInformation;
 }
 
+export interface InspectResultData {
```

Comment:
If this type is specific to `INSPECT`, it shouldn't be inside `service_api_client.ts`.